com.github.arshubham.cipher.json

```diff
@@ -0,0 +1,35 @@
+{
+    "app-id": "com.github.arshubham.cipher",
+    "runtime": "org.gnome.Platform",
+    "runtime-version" : "3.36",
```

Comment:
```suggestion
    "runtime-version" : "3.38",
```

Reply:
I'm really new to this, thanks for helping me out